api/config.js

```diff
@@ -63,10 +63,16 @@ config.auth = {};
 config.database = {};
 config.database.primary = { pool: {} };
 config.mail = {};
+config.token = { expiration: {} };
+config.uri = {};
 
 config.isDevelopment = isDevelopment;
 config.secret = secret;
 config.port = process.env.HACKILLINOIS_PORT || 8080;
+config.domain = isDevelopment ? ('http:localhost:' + config.port + '/') : 'https://www.hackillinois.org/';
```

Comment:
The development domain is malformed (should be `http://` instead of `http:`). This will be fixed in `feature/storage-integration`